test/SampleService_test.py

```diff
@@ -2006,6 +2005,83 @@ def _update_acls(url, token, params, print_resp=False):
     assert ret.json() == {'version': '1.1', 'id': '67', 'result': None}
 
 
+def _update_samples_acls(url, token, params, print_resp=False):
+    resp = requests.post(url, headers=get_authorized_headers(token), json={
+        'method': 'SampleService.update_samples_acls',
+        'version': '1.1',
+        'id': '1729',
+        'params': [params]
+    })
+    if print_resp:
+        print(resp.text)
+    return resp
+
+
+def test_update_acls_many(sample_port):
+    url = f'http://localhost:{sample_port}'
+    # create samples
+    n_samples = 2 # 1000
+    ids = _create_samples(url, TOKEN1, n_samples, 1)
+    for id_ in ids:
+        _update_acls(
+            url,
+            TOKEN1,
+            {
+                'id': str(id_),
+                'admin': [],
+                'write': [],
+                'read': [USER2],
+                'remove': [],
+                'public_read': 1,
+                'as_admin': 0,
+            },
+            print_resp=True,
+        )
+
+
+def test_update_acls_many_bulk(sample_port):
+    url = f'http://localhost:{sample_port}'
+    # create samples
+    n_samples = 2 # 1000
+    ids = _create_samples(url, TOKEN1, n_samples, 1)
+    resp = _update_samples_acls(
+        url,
+        TOKEN1,
+        {
+            'ids': ids,
+            'admin': [],
+            'write': [],
+            'read': [USER2],
+            'remove': [],
+            'public_read': 1,
+            'as_admin': 0,
+        },
+        print_resp=True,
+    )
+    assert resp.ok
+    assert resp.json()['result'] is None
```

Comment:
This also doesn't appear to check that the acls were actually updated correctly.